Classes/BackEnd/AbstractEventMailForm.php

```diff
@@ -4,11 +4,14 @@
 
 namespace OliverKlee\Seminars\BackEnd;
 
+use OliverKlee\Oelib\Email\GeneralEmailRole;
+use OliverKlee\Oelib\Email\SystemEmailFromBuilder;
 use TYPO3\CMS\Backend\Routing\UriBuilder;
 use TYPO3\CMS\Core\Authentication\BackendUserAuthentication;
 use TYPO3\CMS\Core\Messaging\FlashMessage;
 use TYPO3\CMS\Core\Messaging\FlashMessageService;
 use TYPO3\CMS\Core\Utility\GeneralUtility;
+use TYPO3\CMS\Core\Utility\MailUtility;
```

Comment:
This import is not needed anymore. Could you please go through your changes and remove all obsolete imports?